aurora/atomic.py

```diff
@@ -1071,7 +1071,7 @@ def impurity_brems(nz, ne, Te, freq='all', cutoff=0.1):
     # plasma frequency (divide by 2pi to have units of Hz)
     fp = np.sqrt(1e6*ne*constants.e**2/(constants.epsilon_0*constants.m_e))/(2.*np.pi)
 
-    freqT = np.tile(freq, (ne.shape[1], ne.shape[0], 1)).T
+    freqT = np.tile(freq, (ne.shape[0], 1)).T
```

Comment:
@abhatt03 thanks for suggesting these changes. However, they don't seem to be in agreement with the array shapes expected by this function, as described in the function docsstring. For example, `ne` is expected to be both a function of space and time. If you are trying to use this function with `ne` being only a function of radius, I'd suggest giving `ne[None]` (or similar) as input, rather than just `ne` -- then it should work. The `np.ravel(Z)` modification above seems related to this same matter of including time dependence in array shapes.

Reply:
@fsciortino : So I took your suggestion and I began declaring all the arrays in the Aurora namelist as Y[:,None] say for e.g., ne. I am attaching a screenshot to describe the error I receive in such case.

![Error_ne_dimension_Aurora1](https://user-images.githubusercontent.com/92543947/166132699-16be5ca9-d605-4bcf-9c02-510e5769e46d.png)